actors/builtin/miner/miner_actor.go

```diff
@@ -630,19 +630,27 @@ func (a Actor) DeclareTemporaryFaults(rt Runtime, params *DeclareTemporaryFaults
 // Pledge Collateral //
 ///////////////////////
 
-func (a Actor) AwardReward(rt Runtime, _ *adt.EmptyValue) *adt.EmptyValue {
-	rt.ValidateImmediateCallerIs(builtin.RewardActorAddr)
-	pledgeAmount := rt.Message().ValueReceived()
+func (a Actor) AddLockedFund(rt Runtime, amountToLock *abi.TokenAmount) *adt.EmptyValue {
+	rt.ValidateImmediateCallerType(builtin.CallerTypesSignable...)
```

Comment:
This is wrong because the reward actor is not a signable. The strictest thing that might work here is to allow the owner/worker and reward actor. I'll do that.